open3d_slam_ros/include/open3d_slam_ros/Parameters.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Parameters.hpp
+ *
+ *  Created on: Aug 24, 2022
+ *      Author: lukaszpi
+ */
+
+#pragma once
+#include <yaml-cpp/yaml.h>
+#include <string>
+#include <iostream>
+
+#include <geometry_msgs/Point.h>
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/Quaternion.h>
+#include "open3d_slam/Parameters.hpp"
+
+namespace o3d_slam {
+
+
+
+struct MapInitializingParameters {
+  std::string frameId_ = "";
+  geometry_msgs::Pose initialMarkerPose_;
```

Comment:
Some of the ROS dependent stuff like interactive marker parameters are okay to stay here.